engine/src/main/java/com/redhat/cloud/notifications/exports/ExportsService.java

```diff
@@ -0,0 +1,91 @@
+package com.redhat.cloud.notifications.exports;
+
+import com.redhat.cloud.notifications.Constants;
+import org.eclipse.microprofile.faulttolerance.Retry;
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+import org.jboss.resteasy.reactive.RestPath;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.MediaType;
+import java.util.UUID;
+
+@RegisterRestClient(configKey = "export-service")
+public interface ExportsService {
+
+    /**
+     * Sends a JSON payload to the export service.
+     * @param xRhExportServicePsk the export service PSK required for the authorization.
+     * @param exportRequestUuid the {@link UUID} of the export request.
+     * @param application the application the export request got requested to.
+     * @param resourceUuid the {@link UUID} of the requested resource.
+     * @param exportContents the payload of the request.
+     */
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/app/export/v1/{exportRequestUuid}/{application}/{resourceUuid}/upload")
```

Comment:
How does the export service deal with several apps with the same name but from different bundles? (like `advisor`)

Maybe it's only available for the `rhel` bundle.

Reply:
I just understood who `application` is and that's us. Please ignore this comment 😄